src/OpenTelemetry/README.md

```diff
@@ -84,6 +84,44 @@ using var otel = Sdk.CreateTracerProvider(b => b
   * [Building your own Processor](../../docs/trace/extending-the-sdk/README.md#processor)
   * [Building your own Sampler](../../docs/trace/extending-the-sdk/README.md#sampler)
 
+### Troubleshooting
+
+Self diagnostics module is provided to help troubleshooting.
+
+To enable self diagnostics, go to the current directory of your process and
+create a configuration file named `DiagnosticsConfiguration.json` with the
+following content:
+
+```json
+{
+    "LogDirectory": ".",
+    "FileSize": 1024,
+    "EventLevel": "Error"
+}
+```
+
+`LogDirectory` is the directory of the output file. It can be an absolute path
+or a relative path to the current directory. `FileSize` is a positive integer,
+which specifies the log file size in
+[KiB](https://en.wikipedia.org/wiki/Kibibyte).
+[`EventLevel`](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel)
```

Comment:
Would you help to clarify what's the semantic here? (e.g. if setting to `Error`, does this mean anything <= EventLevel.Error will be captured? What about `LogAlways`?)

Reply:
Is this a required config or optional? If it is optional, what's the default behavior?

Reply:
The level signifies the severity of an event. Lower severity levels encompass higher severity levels. For example, Warning includes the Error and Critical levels, which are higher in severity. (from https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel)

Updated in the markdown.

It is required. If missing or of invalid value, the configuration file will be treated as invalid.